SConstruct

```diff
@@ -136,9 +136,8 @@ env = Environment(
     "-fPIC",
     "-O2",
     "-Wunused",
-    "-Werror",
     "-Wno-unknown-warning-option",
-    "-Wno-deprecated-register",
+    "-Wno-deprecated",
```

Comment:
let's fix this in the MPC SConscript, so we can rebuild MPC without these hacks in the main SConsctruct

Reply:
sounds good